libs/ButeMgr/butemgr.h

```diff
@@ -668,8 +669,10 @@ inline bool CButeMgr::Parse(void* pData, unsigned long size, int decryptCode,
 {
 	if (!pData)
 		return false;
+	unsigned int len = size;
+	char* buf1 = (char*)pData;
 #if _MSC_VER >= 1300 && !defined(__clang__)
-	std::istrstream IStream{(char*)pData, size};
```

Comment:
what errors does the compiler give you with this?
maybe make another section for _MSC_VER >= 1500 ?